l10n/css.json

```diff
@@ -1429,5 +1429,17 @@
     "de": "Erstellt <a href=\"/de/docs/Web/CSS/CSS_Positioning/z_index_verstehen/Der_Stackingkontext\">Stapelkontext</a>",
     "fr": "Crée un <a href=\"/fr/docs/Web/CSS/Comprendre_z-index/L'empilement_de_couches\">contexte d'empilement</a>",
     "ru": "Создаёт <a href=\"/ru/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context\">контекст наложения</a>"
+  },
+  "AsAngleBasicshapeOrPath": {
+    "en-US": "as &lt;angle&gt;, &lt;basic-shape&gt; or &lt;path()&gt;",
+    "de": "wie &lt;angle&gt;, &lt;basic-shape&gt; oder &lt;path()&gt;",
```

Comment:
"wie" sounds strange in this context, "als" is better. Having said that, it's ok to leave out the translations and just add the English one.

Sebastian